packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/TransactionsPendingRow.scss

```diff
@@ -41,43 +25,62 @@
         }
 
         &-amount {
-            color: var(--status-light-danger, #ec3f3f);
-            text-align: right;
-
-            /* desktop/paragraph/P2 - bold */
-            font-size: 1.4rem;
-            font-style: normal;
-            font-weight: 700;
-            line-height: 2rem; /* 142.857% */
+            width: max-content;
+        }
 
-            &__deposit {
-                color: var(--status-light-success, #4bb4b3);
-            }
+        &-confirmations {
+            width: 7.6rem;
         }
 
         &-hash {
-            width: 9.8rem;
+            width: 9.2rem;
         }
 
         &-status {
+            width: 10.3rem;
             display: flex;
             justify-content: flex-end;
-        }
+            align-items: center;
+
+            &__dot {
```

Comment:
its recommended for BEM to be flat, 
e.g. instead of:
.SomeComponent__block1__block2

typically its recommended to have:
.SomeComponent__block1
.SomeComponent__block2